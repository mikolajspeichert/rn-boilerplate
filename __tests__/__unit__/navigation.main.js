import 'react-native'
import React from 'react'
import configureStore from 'redux-mock-store'
import { shallow } from 'enzyme'

import Navigator from '@navigation/main'

const state = {
  nav: {
    index: 0,
    routes: [{ routeName: 'Home' }],
  },
}
const mockStore = configureStore()

describe('Main navigator', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Navigator store={mockStore(state)} />)
    expect(wrapper).toMatchSnapshot()
  })
})
