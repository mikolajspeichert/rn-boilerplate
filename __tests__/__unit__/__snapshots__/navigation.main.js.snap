// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main navigator renders correctly 1`] = `
<View
  loaded={
    Array [
      0,
    ]
  }
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
        "overflow": "hidden",
      },
      Object {
        "flex": 1,
      },
    ]
  }
>
  <View
    collapsable={undefined}
    style={
      Object {
        "backgroundColor": "#F7F7F7",
        "borderTopColor": "rgba(0, 0, 0, .3)",
        "borderTopWidth": 0.5,
        "flexDirection": "row",
        "height": 49,
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "rgba(60, 60, 60, 1)",
          "flex": 1,
          "justifyContent": "flex-end",
        }
      }
      testID={undefined}
    >
      <View
        style={
          Object {
            "flexGrow": 1,
          }
        }
      >
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <Image
            source={undefined}
            style={
              Object {
                "height": 26,
                "width": 26,
              }
            }
          />
        </View>
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "opacity": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <Image
            source={undefined}
            style={
              Object {
                "height": 26,
                "width": 26,
              }
            }
          />
        </View>
      </View>
    </View>
  </View>
</View>
`;
